crates/ark/src/lsp/statement_range.rs

```diff
@@ -726,14 +712,15 @@ mod tests {
         let root = ast.root_node();
 
         let node = find_statement_range_node(&root, cursor.unwrap().row).unwrap();
+        let range = expand_range_across_semicolons(node);
 
         assert_eq!(
-            node.start_position(),
+            range.start_point,
             sel_start.unwrap(),
             "Failed on test {original}"
         );
         assert_eq!(
-            node.end_position(),
+            range.end_point,
```

Comment:
Tweaked the test function so we can use `statement_range_test()` even for the new tests